scripts/btgrep.py

```diff
@@ -5,10 +5,13 @@
 import sys
 import urllib
 import urllib2
+import base64
 
-# You need to fill in the domain of where you install
-# Botanist for this to work, obviously :)
-BOTANIST_DOMAIN = 'http://example.com'
+BOTANIST_DOMAIN = 'https://codesearcher.int.sproutsocial.com/'
+# NOTE: Fill out with real username/password to test.
+# (Or come up with a real solution like sticking an auth-less version behind SDM)
+LDAP_USERNAME = ''
```

Comment:
Maybe put these variables in the `.env.template` and that way you can save your `LDAP_USERNAME` and `LDAP_PASSWORD` to the `.env.local` so that there's no chance of an accidental commit...but what could be really fun is integrating it with your SSO provider 😄